cours/containers/docker-orchestration.fr.md

```diff
@@ -124,7 +124,7 @@ docker
 
 ![](images/docker/mesos.png){height="100px"}
```

Comment:
image description would be nice (inside [])

Reply:
sorry that's actually not part of your patch :) but same issue in other parts of your patch